### PR TITLE
issue:#14520 fixed module multiplestringliterals

### PR DESCRIPTION
**Pull request by 555vedant**
issue:#14520 fixed module multiplestringliterals

### CODE REVIEW

**Comment by 555vedant** [comment]
Respected sir , please suggest me changes in my PR?

File: src/xdocs/checks/coding/multiplestringliterals.xml [hunk 1]
```diff
@@ -78,7 +78,7 @@ public class MyClass {
   public void myTest() {
     String a2 = &quot;StringContents&quot;; // violation, &quot;StringContents&quot; occurs twice
     String a3 = &quot;DoubleString&quot; + &quot;DoubleString&quot;; // violation, &quot;DoubleString&quot; occurs twice
-    String a4 = &quot;SingleString&quot;; // OK
```
Comment (MANISH-K-07) [inline src/xdocs/checks/coding/multiplestringliterals.xml:81]: @555vedant ,
Sorry but you still don't seem to understand the requirement of the issue.

The suppression you have removed is related  to the java input file, so please find the respective path.

**Review by MANISH-K-07 — COMMENTED** [review]
Please try to reuse PR instead of closing and creating new PR.

Make sure that `mvn clean verify` passes on your local before pushing changes to remote.
This is mandatory !!

**Comment by MANISH-K-07** [comment]
> @MANISH-K-07 please give me source of prev merged PR's ,sorry for my work but i am new to this organization and i am trying to understand things..

Please check my comment at https://github.com/checkstyle/checkstyle/pull/14520#pullrequestreview-1897326458 for Examples of updates

**Comment by 555vedant** [comment]
> should we have to purchase Cirrus - JDK17 ?

what about this ?

File: config/checkstyle-input-suppressions.xml [hunk 2]
```diff
@@ -68,8 +68,6 @@
           files="[\\/]packageobjectfactory[\\/]zoo[\\/]FooCheck.java"/>
 
   <!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]multiplestringliterals[\\/]InputMultipleStringLiteralsNoWarnings.java"/>
```
Comment (nrmancuso) [inline config/checkstyle-input-suppressions.xml:72]: Remove `// ok` comments from this exact java file.

**Pull request closed without merging**